src/scripts/main.js

```diff
@@ -1,3 +1,75 @@
 'use strict';
 
 // write code here
+let rowCount = 4;
+let columnCount = 4;
+const maxCount = 10;
+const minCount = 2;
```

Comment:
These variables define the initial, maximum, and minimum counts for rows and columns. Ensure that the table is initialized with `rowCount` rows and `columnCount` columns to avoid any unexpected behavior when interacting with the buttons.